data.py

```diff
@@ -2,14 +2,14 @@
 from models import db
 
 
-def statewide_by_day(current_days_before):
+def statewide_by_day():
     sql = """
         SELECT * FROM cumulative_stats_by_day
-        WHERE days_before < 45 AND days_before > :current_days_before;
+        WHERE days_before < 45 AND days_before >= (select '2021-01-05' - max(date(file_update_time)) from updated_times);
```

Comment:
Okay I think this is a way to address the problem i mentioned. This will make it so that if the last date of data is not between 6pm-11:59pm, it will skip that day.

```suggestion
        WHERE days_before < 45 AND days_before >= (select '2021-01-05' - max(date(file_update_time - interval '18 hours')) from updated_times);
```

Here is how the last few data updates would get rounded off:
```
 2020-12-10 23:54:06 | 2020-12-10
 2020-12-12 00:02:50 | 2020-12-11
 2020-12-13 00:01:04 | 2020-12-12
 2020-12-13 23:36:58 | 2020-12-13
 2020-12-15 00:04:34 | 2020-12-14
 2020-12-16 00:13:16 | 2020-12-15
```